hackathon_site/dashboard/frontend/src/testing/mockData.tsx

```diff
@@ -16,6 +16,75 @@ import { CartItem } from "api/types";
 import { adminGroup } from "constants.js";
 
 // For DashCard on Dashboard
+
+export const mockOverviewStats = [
```

Comment:
I feel like the api for this will be very specific something like this:
```
{
  checkedOutOrders: <number>,
  numUsers: <number>,
  etc.
}
```
So there's no need for a general array structure